src/App.css

```diff
@@ -1,39 +1,64 @@
 .App {
   text-align: center;
+  font-size: small;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.calculator {
+  display: flex;
+  background: whitesmoke;
+  flex-direction: column;
+  width: 300px;
+  height: 280px;
+  position: absolute;
+  right: 40vw;
+  top: 10vw;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.number-screen {
   display: flex;
-  flex-direction: column;
+  background: grey;
+  width: 100%;
+  height: 17%;
+  justify-content: flex-end;
   align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
+  text-align: center;
   color: white;
+  margin-top: 2px;
 }
 
-.App-link {
-  color: #61dafb;
+.input {
```

Comment:
- [ ] The design is not matching with the given design. Please fix that.

### Given design
![Screenshot from 2023-02-15 01-08-58](https://user-images.githubusercontent.com/26767383/218840244-a553d6c1-1f4e-466b-ade8-a75911f70208.png)


### Your design
![Screenshot from 2023-02-15 01-09-11](https://user-images.githubusercontent.com/26767383/218840316-58166132-e784-4ab0-a498-e4691723c41f.png)

Reply:
Thank you very much for the thorough review, changes are being implemented